tests/admin_views/tests.py

```diff
@@ -1480,6 +1507,24 @@ def test_login(self):
         form = login.context[0].get('form')
         self.assertEqual(form.errors['username'][0], 'This field is required.')
 
+    def test_login_view_user(self):
+        """
+        Make sure only staff members can log in.
```

Comment:
This docstring should be customized for the purpose of this test (it looks to me like it's copied from the other test).